$OpenBSD$
We need this patch because leak of really static boostrapper. As the rustc
file in bootstrap is a shell script (see post-extract), the sysroot is badly
infered. Use this patch to always have the good sysroot during build.
--- mk/main.mk.orig	Wed Feb 18 13:56:24 2015
+++ mk/main.mk	Thu Feb 19 08:58:56 2015
@@ -415,7 +415,7 @@ CFG_VALGRIND_COMPILE$(1) = $$(CFG_VALGRIND_COMPILE)
 endif
 
 # Add RUSTFLAGS_STAGEN values to the build command
-EXTRAFLAGS_STAGE$(1) = $$(RUSTFLAGS_STAGE$(1))
+EXTRAFLAGS_STAGE$(1) = --sysroot $$(HROOT$(1)_H_$(3)) $$(RUSTFLAGS_STAGE$(1))
 
 CFGFLAG$(1)_T_$(2)_H_$(3) = stage$(1)
 
